### PR TITLE
Finished page layout

### PR DESCRIPTION
**Pull request by lucas-shugg**

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import 'package:flutter/material.dart';
 
 void main() {
-  runApp(const MyApp());
+  runApp(const MyApp(key: Key('value')));
```
Comment (ted-marozzi) [inline lib/main.dart]: Dont need a key here, keys are optional and usually used for testing,

File: lib/main.dart [hunk 2]
```diff
@@ -36,6 +36,250 @@ class MyHomePage extends StatelessWidget {
...
+                        width: 100,
+                        height: 70,
+                        child: Text(
+                          "MENU",
+                          style: Theme.of(context)
+                              .textTheme
+                              .headline5!
+                              .copyWith(
+                                  fontFamily: 'Impact',
+                                  color: const Color.fromRGBO(242, 11, 249, 1)),
```
Comment (ted-marozzi) [inline lib/main.dart]: This should be a variable as its used multiple times

File: lib/main.dart [hunk 3]
```diff
@@ -36,6 +36,250 @@ class MyHomePage extends StatelessWidget {
...
+                  children: [
+                    SizedBox(
+                      width: 900,
+                      height: 100,
+                      child: Text(
+                        "EGP GROWTH ETF PORTFOLIO",
+                        textAlign: TextAlign.center,
+                        style: Theme.of(context).textTheme.headline2!.copyWith(
+                            fontFamily: 'Impact',
+                            color: const Color.fromRGBO(242, 11, 249, 1)),
```
Comment (ted-marozzi) [inline lib/main.dart]: like here

File: lib/main.dart [hunk 4]
```diff
@@ -36,6 +36,250 @@ class MyHomePage extends StatelessWidget {
...
+                  ]),
+              Row(
+                  crossAxisAlignment: CrossAxisAlignment.center,
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: [
+                    SizedBox(
+                      width: 1500,
+                      height: 100,
+                      child: Text(
+                        "EGP's Growth ETF Portfolio has been constructed as an educational resource. The aim is to highlight the importance of diversification, provide examples of ETFs, and to help you understand asset classes and what they mean.",
```
Comment (ted-marozzi) [inline lib/main.dart:106]: EGP -> P stands for portfolio so change it to EG or explicitly say Equity Gals

File: lib/main.dart (hunk 1 above)
Comment (lucas-shugg) [reply lib/main.dart]: G'day noob, thanks for the feedback, will resolve his right away

File: lib/main.dart [hunk 5]
```diff
@@ -36,6 +36,250 @@ class MyHomePage extends StatelessWidget {
...
+                    ),
+                  ]),
+              Row(
+                  crossAxisAlignment: CrossAxisAlignment.center,
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: [
+                    Container(
+                      height: 50,
+                      width: 300,
+                      color: const Color.fromRGBO(56, 88, 78, 1),
```
Comment (ted-marozzi) [inline lib/main.dart]: Should be a variable

File: lib/main.dart (hunk 2 above)
Comment (lucas-shugg) [reply lib/main.dart]: G'day noob, I have fixed all of these doubled variables and have created our own.

**Pull request merged**